Gemfile

```diff
@@ -65,6 +65,9 @@ gem 'sidekiq-cronitor', '~> 3.6.0'
 # HTTP requests
 gem 'httparty', '~> 0.21.0'
 
+# FIXME(ezekg) Hack: https://github.com/jnunemaker/httparty/issues/797
+gem 'csv'
```

Comment:
We need to replace HTTParty with [`http.rb`](https://github.com/httprb/http) anyways. But that can be done in another PR.

Reply:
I kind of want to do this before we upgrade to Ruby v3.3. I don't like hacks.

Reply:
No longer needed https://github.com/jnunemaker/httparty/issues/797.